packages/fx-core/src/plugins/resource/aad/v2/index.ts

```diff
@@ -52,7 +52,7 @@ export class AadPluginV2 implements v2.ResourcePlugin {
     inputs: v2.ProvisionInputs,
     envInfo: Readonly<v2.EnvInfoV2>,
     tokenProvider: TokenProvider
-  ): Promise<Result<v2.ResourceProvisionOutput, FxError>> {
+  ): Promise<Result<Json, FxError>> {
```

Comment:
Need to change to void.
After offline discussion, @jayzhang will fix in another pr.